### PR TITLE
Add KerbalFuels.netkan

### PR DESCRIPTION
**Pull request by plague006**
Breakup of #1786

### CODE REVIEW

File: NetKAN/KerbalFuels.netkan [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "spec_version": 1,
+    "license": "CC BY-NC-SA 4.0",
```
Comment (dbent) [inline NetKAN/KerbalFuels.netkan]: `CC-BY-NC-SA-4.0`

**Comment by plague006** [comment]
Corrected

**Comment by dbent** [comment]
Missed the space at the end. :wink:

**Comment by plague006** [comment]
lol, I lost my test environment when a drive failed, I was relying on jenkins for catching the small things :(

**Comment by dbent** [comment]
Tested, manually. Merging.

Apparently even a butler like Jenkins needs a vacation every now and then.

**Pull request merged**